### PR TITLE
ENG-16236 tracing tool benchmark test case

### PR DESCRIPTION
**Pull request by victorxgao**

### CODE REVIEW

File: tests/test_apps/voter-tracing-benchmark/README [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+To choose "DistinctCount", set "-–doInsert=false" and "--doSelectCount=true"
+
+(2) To choose which tracing subsystems (such as CI, SPI, EE, MPI etc.) to turned on, 
+the user needs to set related commands in function tracing-benchmark-figurePlot() 
+and/or function tracing-benchmark-showAll () in the bash script run.sh. 
+
+For example, to turned on the CI subsystem before test, one need to do like this:
+
+sqlcmd --query="exec @Trace enable CI" > /dev/null
+#doing tests
```
Comment (rdykiel) [inline tests/test_apps/voter-tracing-benchmark/README]: "run the tests"

File: tests/test_apps/voter-tracing-benchmark/README [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+either of them by setting the parameters in the function tracing-benchmark() in run.sh.
+
+To choose "VOTES.INSERT", set set "-–doInsert=true" and "--doSelectCount=false"
+To choose "DistinctCount", set "-–doInsert=false" and "--doSelectCount=true"
+
+(2) To choose which tracing subsystems (such as CI, SPI, EE, MPI etc.) to turned on, 
+the user needs to set related commands in function tracing-benchmark-figurePlot() 
+and/or function tracing-benchmark-showAll () in the bash script run.sh. 
+
+For example, to turned on the CI subsystem before test, one need to do like this:
```
Comment (rdykiel) [inline tests/test_apps/voter-tracing-benchmark/README]: "to turn on" or "to enable"

File: tests/test_apps/voter-tracing-benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+--------------------------------------------------------------------------------
+Brief Introduction
+--------------------------------------------------------------------------------
+To measure performance impact of VoltDB tracing system, a case study example is
```
Comment (rdykiel) [inline tests/test_apps/voter-tracing-benchmark/README.md]: "of the VoltDB..."

File: tests/test_apps/voter-tracing-benchmark/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+--------------------------------------------------------------------------------
+Notes
+--------------------------------------------------------------------------------
+(1) As mentioned before, there are two stored procedures "VOTES.INSERT" and "DistinctCount". 
+In the example, the default stored procedure is "VOTES.INSERT". The user can choose 
+either of them by setting the parameters in the function tracing-benchmark() in run.sh.
+
+To choose "VOTES.INSERT", set set "-–doInsert=true" and "--doSelectCount=false"
+To choose "DistinctCount", set "-–doInsert=false" and "--doSelectCount=true"
```
Comment (rdykiel) [inline tests/test_apps/voter-tracing-benchmark/README.md]: What happens if you choolse "true" or "false" on both?

File: tests/test_apps/voter-tracing-benchmark/README.md (hunk 3 above)
Comment (victorxgao) [reply tests/test_apps/voter-tracing-benchmark/README.md]: README modified.Thank you!

File: tests/test_apps/voter-tracing-benchmark/README.md (hunk 4 above)
Comment (victorxgao) [reply tests/test_apps/voter-tracing-benchmark/README.md]: When we set these two parameters to be "true", both "VOTES.INSERT" and "DistinctCount" will be run in the benchmark. If we set them to be "false", neither will be run in the benchmark. This is not what we want. I will modify code. Thanks for pointing out this out.

**Pull request merged**